src/pyubx2/ubxreader.py

```diff
@@ -197,6 +200,7 @@ def _parse_ubx(self, hdr: bytes) -> tuple:
         Parse remainder of UBX message.
 
         :param bytes hdr: UBX header (b'\xb5\x62')
+        :param bool parsing: a boolean flag to determine if the stream needs to be parsed

```

Comment:
can remove this docstring entry as it no longer applies

Reply:
Removed